### PR TITLE
Monitor title color

### PR DESCRIPTION
**Pull request by Nailim**
Another feature for your consideration, added the ability to display monitor title in color depending on computer load (default colors used in the pop-up or custom specified).

This allows to have a less distracting option on display (governer or load) and still see the status of the computer. Similar wish was expressed in this feature request: https://github.com/konkor/cpufreq/issues/81

### CODE REVIEW

**Comment by konkor** [comment]
You done a lot changes. I'm testing it now and have a few thoughts about it:
1. Style states should be more general not CPU loading focused like normal, warning and critical (I want to add thermal throttle monitoring so)
2. The design of color preferences doesn't look finished. I would replaced Gtk.Entry with Gtk.ColorButton picker. Also I would add some interactivity, switching a sensitivity of widgets.
3. I'm not sure about additional signal for the style changing. It could be just second parameter on basic signal.

I'll make some changes and check it more if you don't mind.

Thanks for your work!

File: cpufreq-service [hunk 1]
```diff
@@ -220,6 +258,13 @@ var CpufreqService = new Lang.Class ({
             this.old = this.title;
             this.max0 = m;
         }
+        if (color_show) {
+          s = this.stylestate;
```
Comment (konkor) [inline cpufreq-service]: this.stylestate is always undefined

**Comment by konkor** [comment]
Okay merged to Master. I done some changes so.

**Comment by Nailim** [comment]
Awesome. Cheers also on the tweaks!

**Pull request merged**